components/developer/valgrind/Makefile

```diff
@@ -22,18 +22,19 @@
 #
 # Copyright (c) 2015-2017 Aurelien Larcher. All rights reserved.
 # Copyright (c) 2019 Michal Nowak
+# Copyright (c) 2021 David Stes
 #
 
```

Comment:
While you are at it:
Can you please also update to the newer
BUILD_BITS= 64
here and later...